### PR TITLE
Operations and tests on collections

### PR DESCRIPTION
**Pull request by pcm32**
This PR accompanies https://github.com/ebi-gene-expression-group/atlas-schemas/pull/9 to add operation and testing on top of collections schema.

### CODE REVIEW

**Review by pinin4fjords — APPROVED** [review]
Looks good to me- nice clean code!

File: bin/create_collection.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Check that necessary environment variables are defined.
+[ -z ${dbConnection+x} ] && echo "Env var dbConnection for the database connection needs to be defined. This includes the database name." && exit 1
+[ -z ${COLL_ID+x} ] && echo "Env var COLL_ID for the id of the collection needs to be defined." && exit 1
+[ -z ${COLL_NAME+x} ] && echo "Env var COLL_NAME for the name of the experiment needs to be defined." && exit 1
+
+ICON_IMPORT=""
+ICON_INSERT_FIELD=""
+ICON_INSERT_VALUE=""
+echo $( pwd )
```
Comment (pcm32) [inline bin/create_collection.sh]: ohh, this escaped! was part of my debugging!

Comment (pcm32) [inline bin/create_collection.sh]:
```suggestion
```

**Comment by pcm32** [comment]
Given the review and the fact that @anjaf is happy with the diversity of operations, I'll merge this. Thanks guys.

**Pull request merged**